.npmrc

```diff
@@ -1,2 +1,3 @@
 package-lock=true
 registry="https://registry.npmjs.org"
+legacy-peer-deps=true
```

Comment:
is this needed?

Reply:
After clearing the package-locks and attempting to reinstall it would throw an error because of [ember-power-calendar-selectors](https://github.com/yahoo/ember-power-calendar-selectors/blob/089ec06b4930e08c869f9fdd04aae1f50542358b/package.json#L66), but we could probably go without committing it we just need to update that library